packages/lesswrong/components/tagging/FooterTagList.tsx

```diff
@@ -10,7 +10,7 @@ import { useTracking } from "../../lib/analyticsEvents";
 const styles = theme => ({
   root: {
     marginTop: 16,
-    marginBottom: 16,
+    marginBottom: 66,
```

Comment:
What is happening here? Either it's an error, or I will notice how this makes sense when I actually boot this up, but does sure feel a bit doomy.

Reply:
I checked the UI for this and somehow it didn't look weird. I didn't check how it compared to devel though.

Reply:
Ah, cool. I get why it's here now. It replaced the bottom margin for the postBody.

However, in order to just preserve the previous spacing, the correct number is `50px.`

```suggestion
    marginBottom: 50,
```